### PR TITLE
record list type

### LINKED ISSUE #137: [UI] allow show list of text as record inputs.
Screenshot 2021-06-14 at 11 31 03

### PR DESCRIPTION
**Pull request by leiyre**
Allow record list type

Closes #137

### CODE REVIEW

**Comment by frascuchon** [comment]
@leiyre  closes this PR the issue #137 ?

File: frontend/components/text-classifier/RecordInputs.vue [hunk 1]
```diff
@@ -28,8 +29,20 @@ export default {
...
+      if (typeof (record) === 'string' || typeof (record) === 'number') {
+        return true;
+      }
+      return false;
+    },
+    isList(record) {
+      if (Array.isArray(record)) {
+        return true;
+      }
+      return false;
```
Comment (frascuchon) [inline frontend/components/text-classifier/RecordInputs.vue]:
```suggestion
      return Array.isArray(record);
```

**Comment by frascuchon** [comment]
`RecordList.vue` and `RecordString.vue` are almost identical: only changes the record `span` section.

I feel more confortable including changes in `RecordString.vue` that enables the list visualization.

This simplifies components maintainability

**Comment by leiyre** [comment]
@frascuchon yes, this PR fix #137


> `RecordList.vue` and `RecordString.vue` are almost identical: only changes the record `span` section.
> 
> I feel more confortable including changes in `RecordString.vue` that enables the list visualization.
> 
> This simplifies components maintainability

I know, components refactor is needed to avoid duplicate code, I'm looking for the best way

**Comment by frascuchon** [comment]
I've done some tests and I think we should manage multiple elements in list. Here an record with 4 element in text list

```yaml
inputs.text: 
   0: "Esto es un input colega"
   1: "otro"
   2: "que pasa tronco"
   3: "nada de nada"
```
Screenshot 2021-06-15 at 14 00 20

**Review by frascuchon — APPROVED** [review]

**Pull request merged**